pbf/views.py

```diff
@@ -1136,6 +1138,16 @@ def ready(request, player_id):
     compute_card_thresholds(player)
     return with_log_trigger(render(request, 'player.html', {'player': player}))
 
+def add_impending_log_msgs(player):
+    for card in player.impending_with_energy.all():
+        impended_card_with_energy = get_object_or_404(GamePlayerImpendingWithEnergy, gameplayer=player, card=card)
```

Comment:
for performance reasons, this should instead be:

```suggestion
    for impended_card_with_energy in player.gameplayerimpendingwithenergy_set.all():
```

we can compare what happens when we enable the database logging in settings.py https://github.com/nathanj/spirit-island-pbp/blob/a4dfc52f1ebc11a9293ac5ee621994b939f77571/island/settings.py#L153-L173

before this change: does an inner join for the cards, then re-queries the join table once for each individual GamePlayerImpendingWithEnergy
after this change: queries the join table once total, getting the relevant GamePlayerImpendingWithEnergy objects